src/modules/__test__/add-remove.test.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { listen, addToLi } from '../addtask.mjs';
+import remove from '../remove.mjs';
+
+// Add Test
+
+describe('addToLi Function', () => {
+  test('add Li element', () => {
+    document.body.innerHTML = '<ul id="todoList"></li>';
```

Comment:
- [ ] I believe you want to create `<li>` tag here, if so, kindly add the beginning `<li>` tag instead of `<ul>`.

Reply:
Oh, yes there is a little mistake but it needs to be a `<ul>` element. Needs to be:
> `<ul id="todoList"></ul>`
We will correct that